examples/nxp/rt1060-evk-make-baremetal-builtin/main.c

```diff
@@ -65,6 +107,6 @@ int main(void) {
   for (;;) {
     mg_mgr_poll(&mgr, 0);
   }
-
```

Comment:
Why ? All examples have this line, it separates the loop from the (useless) rest